Riot/Modules/Pills/PillTextAttachmentData.swift

```diff
@@ -93,12 +95,14 @@ struct PillTextAttachmentData: Codable {
          items: [PillTextAttachmentItem],
          isHighlighted: Bool,
          alpha: CGFloat,
-         font: UIFont) {
+         font: UIFont,
+         maxWidth: CGFloat = CGFloat.greatestFiniteMagnitude) {
```

Comment:
Should be able to infer the type of the default here :)
```suggestion
         maxWidth: CGFloat = .greatestFiniteMagnitude) {
```